modules/ROOT/pages/errors/index.adoc

```diff
@@ -1,91 +1,208 @@
-// | 2D000	        | error: invalid transaction termination
-// | 40000	        | error: transaction rollback
-// | 40003	        | error: transaction rollback - statement completion unknown
-// | 42000	        | error: syntax error or access rule violation
-// | 42001	        | error: syntax error or access rule violation - invalid syntax
-// | 42002	        | error: syntax error or access rule violation - invalid reference
-// | 42004	        | error: syntax error or access rule violation - use of visually confusable identifiers
-// | 42006	        | error: syntax error or access rule violation - number of edge labels below supported minimum
-// | 42007	        | error: syntax error or access rule violation - number of edge labels exceeds supported maximum
-// | 42008	        | error: syntax error or access rule violation - number of edge properties exceeds supported maximum
-// | 42009	        | error: syntax error or access rule violation - number of node labels below supported minimum
-// | 42010	        | error: syntax error or access rule violation - number of node labels exceeds supported maximum
-// | 42011	        | error: syntax error or access rule violation - number of node properties exceeds supported maximum
-// | 42012	        | error: syntax error or access rule violation - number of node type key labels below supported minimum
-// | 42013	        | error: syntax error or access rule violation - number of node type key labels exceeds supported maximum
-// | 42014	        | error: syntax error or access rule violation - number of edge type key labels below supported minimum
-// | 42015	        | error: syntax error or access rule violation - number of edge type key labels exceeds supported maximum
-// | G1000	        | error: dependent object error
-// | G1001	        | error: dependent object error - edges still exist
-// | G1002	        | error: dependent object error - endpoint node is deleted
-// | G1003	        | error: dependent object error - endpoint node not in current working graph
-// | G2000	        | error: graph type violation
-// |===
-
-// [[gqlstatus-neo4j-defined-codes]]
-// === Neo4j-defined GQLSTATUS error codes
+The Neo4j DBMS returns a server error to indicate that the outcome of a Cypher query or command execution is unsuccessful.
+The driver receives these erors and sends them to the Neo4j tools (e.g. Browser, Bloom, Cypher Shell) or the user application, which display them to the user.
+
+Starting from version 5.26, the Neo4j error codes have an additional GQL-status object along with the Neo4j exception that provides information about the status of a Cypher query or command execution in compliance with the link:https://www.iso.org/standard/76120.html[ISO/IEC 39075:2024(en) - Information technology - Database languages - GQL Standard].
+This additional GQL-status object is also displayed in the query log from Neo4j 5.25.
```

Comment:
"... from Neo4j 5.25"

Is here a word missing? Like "... from Neo4j 5.25 onwards" or "... starting from Neo4j 5.25" or such?